### PR TITLE
Tejpal branch

### PR DESCRIPTION
**Pull request by tejpalDarro**

### CODE REVIEW

File: tejpalFolder/Assignment1/questions.txt [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+------------------------------------------------------------------------------------------
+
+Que. What is the command for installing Git on your Linux system?
+-You can install Git on Linux through the package management tool that comes with your distribution.
+
+Linux
+Git packages are available using apt.
+It's a good idea to make sure you're running the latest version. To do so, Navigate to your command
+prompt shell and run the following command to make sure everything is up-to-date: sudo apt-get update.
+To install Git, run the following command: sudo apt-get install git-all.
```
Comment (Pawan1005) [inline tejpalFolder/Assignment1/questions.txt:31]: remove -all from cmd

File: tejpalFolder/Assignment1/questions.txt [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+Once the command output has completed, you can verify the installation by typing: git version.
+------------------------------------------------------------------------------------------
+
+Que. Why should you set up user.name and user.email configuration? How do you set them up?
+-When you make commits on your local system and push them to Bitbucket Cloud, the commit data determines 
+which account name to attach to the push.
+
+To ensure your commits in Bitbucket appear with your user account, it must meet two conditions:
+
+You've configured Git with a global username/email address and an optional repository-specific username/email
```
Comment (Pawan1005) [inline tejpalFolder/Assignment1/questions.txt:45]: add proper command for global user/name

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: tejpalFolder/Assignment1/questions.txt (hunk 2 above)
Comment (tejpalDarro) [reply tejpalFolder/Assignment1/questions.txt:45]: done.

File: tejpalFolder/Assignment1/questions.txt (hunk 1 above)
Comment (tejpalDarro) [reply tejpalFolder/Assignment1/questions.txt:31]: done.

**Pull request merged**